packages/next-urql/src/with-urql-client.tsx

```diff
@@ -46,7 +57,17 @@ export function withUrqlClient(clientConfig: NextUrqlClientConfig) {
         : (appOrPageCtx as NextPageContext);
 
       const ssrCache = ssrExchange({ initialState: undefined });
-      const urqlClient = initUrqlClient(clientConfig(ssrCache, ctx));
+      const client = clientConfig(ssrCache, ctx);
```

Comment:
this naming is a little confusing 😅  let's maybe keep calling this `clientConfig` and replace the function with `get*` or `make*`?